language/stdlib/modules/LibraAccount.move

```diff
@@ -1121,6 +1198,14 @@ module LibraAccount {
         // Check that the chain ID stored on-chain matches the chain ID specified by the transaction
         assert(ChainId::get() == chain_id, Errors::invalid_argument(PROLOGUE_EBAD_CHAIN_ID));
 
+        // Check that the gas calculation will not overflow
```

Comment:
Why was this check moved?

Reply:
Actually I shouldn't move them. This was coming from my previous version of refactor where I had the gas related function outside of `prologue_common`